.github/workflows/regenerate-newsroom-videos.yml

```diff
@@ -0,0 +1,31 @@
+name: List everyday latest list of Newsroom Videos
+
+on: 
+  workflow_dispatch:
```

Comment:
We probably don't need that (it's empty object.

```suggestion
```

Reply:
@derberg Should I remove this, if it is not making any changes to Action?

Reply:
keep it, we have it in meetings workflow, it allows for a manual run. I used it few times already like this, to reflect on a website something I wanted to have immediately, not waiting for next day

<img width="2213" alt="Screenshot 2022-07-21 at 10 43 41" src="https://user-images.githubusercontent.com/6995927/180171071-e03b2632-2d5c-4077-ac3b-c57e804ae3bd.png">